### DISCUSSION TITLE
No puedo hacer mvn verify

### DISCUSSION THREAD

**Discussion by MartoSimon** [dds-jv/foro#421] [Consultas de herramientas] [answered]
Holaa, estaba intentando corre los comandos de mvn para poder terminar el tp individual pero no me está dejando correrlos. Me aparece siempre este error.

![image](https://github.com/dds-jv/foro/assets/72176653/fa67bec5-2c25-49ec-b5b0-088b14db15ab)

Les copio el error por si les sirve:
mvn : El término 'mvn' no se reconoce como nombre de un cmdlet, función, archivo de script o programa ejecutable. Compruebe si escribió correctamente el 
nombre o, si incluyó una ruta de acceso, compruebe que dicha ruta es correcta e inténtelo de nuevo.
En línea: 1 Carácter: 1
+ mvn test
+ ~~~
    + CategoryInfo          : ObjectNotFound: (mvn:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Tengo instalado maven como muestro en esta foto.
![image](https://github.com/dds-jv/foro/assets/72176653/8af3110e-7923-423b-9490-6b4d3cf029fb)

**Comment by flbulgarelli** [comment]
¡Hola! ¿Desde donde estas corriendo el comando `mvn verify`? ¿El error se está produciendo en la misma terminal que la que usaste para correr `mvn -v`?

**Reply by flbulgarelli** [reply]
Mandamos por favor una captura de pantalla donde se vea el error de `mvn verify` justo debajo de `mvn -v`.

**Comment by MartoSimon** [comment, marked as answer]
Ahi me funciono bien, estaba ejecutando los comandos en 2 consolas por separado. Muchas gracias!
